changelog/seqera-enterprise/v25.1.mdx

```diff
@@ -0,0 +1,144 @@
+- Add support for Fusion licensing.
+
+### General
+
+- Updated list of EC2 families with NVMe disk available 
+- Audit log update: Pipeline edit events are now logged.
+- Switch AWS Batch compute environment dependencies to AWS SDK v2.
+- Switch Compute dependencies to AWS SDK v2.
+- When a GitHub pipeline is added or updated, if no user-uploaded icon is defined, the icon will default to the one defined the `manifest.icon` Nextflow config; otherwise, it will use the GitHub organization avatar. You can also upload custom icons, which take priority over the auto fetched ones.
+- New dynamic page title for easy bookmarking.
+- Added `totalProcesses` to workflow progress responses.
+- Implement collapsible view for JSON workflow parameters tab and add **View as YAML** option.
+- Update the pipeline name regular expression to allow pipeline names containing dots ('.').
+- Allow Nextflow configuration parameters with embedded references to other parameters to be shown verbatim.
+- Improved error messaging when pipeline information can't be fetched indicates whether the issue is due to a missing resource, or failed authentication due to expired credentials.
+- Azure jobs are now automatically terminated after all tasks are complete.
+- Send only added/updated run parameters when launching a pipeline. This includes all defaults and parameters passed during the launch.
+- Allow users to remove organization logo.
+- New workflow job monitoring collects and publishes Platform metrics.
+- Notify the user when secrets cleanup fails.
+- Dashboard page: Add a date filter. 
+- Bump nf-launcher default to version 25.10.5.
+- Upgrade to Angular 18.
+
+## Bug fixes
+
+- **Studios**
+  - Fixed resource labels being erroneously non-editable for the Maintain role when adding or starting a Studio session.
+  - Fixed searches for names containing special SQL wildcard characters (_, %).
+- Prohibit duplicate Git credentials and tie-break on `lastUpdated` for existing duplicates. Going forward, preference is given to the credentials updated last in the situation when there are multiple candidates. Additionally, a duplicate check is added when creating new credentials to avoid new duplicate cases.
+- Set `ECS_CONTAINER_START_TIMEOUT` to 10 minutes in the ECS settings used by AWS Batch CEs in order to prevent the error `Task failed to start - DockerTimeoutError: Could not transition to started; timed out after waiting 3m0s`.
+- Compute environment creation form allows pre- and post-run scripts to be longer than the accepted value of 1024 characters.
+- Disable reset selection on data change in workflow list component.
+- Take into account the alternative `mainScript` path.
+- Use `preferred_username` as fallback email field for OIDC login.
+- Drop the last characters in job definition name, if longer than the prescribed limit.
+- Fixed users not being able to remove the organization logo.
```

Comment:
```suggestion
```
We label this as both a feature under General and as a bug fix. I suggest we remove it from bug fixes.